{{cookiecutter.repostory_name}}/tools/backup-b2/retrieve_backup.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+b2 authorize-account "$BACKUP_B2_KEY_ID" "$BACKUP_B2_KEY_SECRET"
+b2 download-file-by-name "$BACKUP_B2_BUCKET" "$1" ".backups/$(basename "$1")"
+
```

Comment:
how does the user know if the script finished successfully? Could you please add `echo ok`?

Reply:
There's already output from b2 download-file-by-name, and script is not meant to be run directly anyway.